examples/add_pedestal_noise_template.xml

```diff
@@ -0,0 +1,114 @@
+
+    <service id="calibService" class="fact.calibrationservice.ConstantCalibService" />
+    
+    
+    <property name="noiseDB" value="file:${SET_noiseDB}" />
+    <property name="dataFolder" value="${SET_dataFolder}" />
+    <property name="noiseBinning" value="2" />
+
+    <property name="LONS_KEYS" value="LONS_NIGHT,LONS_RUNID,LONS_EventNum,LONS_UnixTimeUTC"/>
+    <property name="MC_KEYS" value="IncidentAngle.fVal,MCorsikaEvtHeader.fEvtNumber,MCorsikaEvtHeader.fNumReuse,MCorsikaEvtHeader.fTotalEnergy,MCorsikaEvtHeader.fStartAltitude,MCorsikaEvtHeader.fFirstTargetNum,CorsikaEvtHeader.fFirstInteractionHeight,MCorsikaEvtHeader.fMomentumX,MCorsikaEvtHeader.fMomentumY,MCorsikaEvtHeader.fMomentumZ,MCorsikaEvtHeader.fZd,MCorsikaEvtHeader.fAz,MCorsikaEvtHeader.fX,MCorsikaEvtHeader.fY,MCorsikaEvtHeader.fWeightedNumPhotons,MMcEvt.fEvtNumber,MMcEvt.fThick0,MMcEvt.fFirstTarget,MMcEvt.fZFirstInteraction,MMcEvt.fCoreD,MMcEvt.fCoreX,MMcEvt.fCoreY,MMcEvt.fTimeFirst,MMcEvt.fTimeLast,MMcEvt.fLongiNmax,MMcEvt.fLongit0,MMcEvt.fLongitmax,MMcEvt.fLongia,MMcEvt.fLongib,MMcEvt.fLongic,MMcEvt.fLongichi2,MMcEvt.fPhotIni,MMcEvt.fPassPhotAtm,MMcEvt.fPassPhotRef,MMcEvt.fPassPhotCone,MMcEvt.fPhotElfromShower,MMcEvt.fPhotElinCamera,MMcEvt.fElecCphFraction,MMcEvt.fMuonCphFraction,MMcEvt.fOtherCphFraction,MMcEvt.fFadcTimeJitter,MMcEvt.fEventReuse,MMcEvtBasic.fEnergy,MMcEvtBasic.fImpact,MMcEvtBasic.fTelescopePhi,MMcEvtBasic.fTelescopeTheta,MMcEvtBasic.fTheta,MMcEvtBasic.fPhi,MPointingPos.fZd,MPointingPos.fAz,MPointingPos.fRa,MPointingPos.fHa,MPointingPos.fDec,MRawEvtData.fHiGainPixId,Data,StartCellData,TriggerType,MRawEvtData.fNumBytesPerSample,MRawEvtData.fIsSigned,EventNum,MRawEvtHeader.fNumTrigLvl1,MRawEvtHeader.fTrigPattern,MSimSourcePos.fZd,MSimSourcePos.fAz,MSimSourcePos.fRa,MSimSourcePos.fHa,MSimSourcePos.fDec,McCherPhotWeight,McCherPhotNumber,McMuonCherPhotWeight,McMuonCherPhotNumber,McNoisePhotWeight,McCherArrTimeMean,McCherArrTimeVar,McCherArrTimeMin,McCherArrTimeMax"/>
+    
+    <property name="OUTPUT_KEYS" value="${MC_KEYS},${LONS_KEYS}"/>
+
+    <stream id="fact" class="fact.io.FactFileListMultiStream"   url="${input}">
+        <stream class="fact.io.hdureader.FITSStream" id="fact" />
+    </stream>
+
+    <process id="1" input="fact">
+        <fact.utils.Remapping
+                key="Data"
+                outputKey="Data"
+        />
+
+        <fact.datacorrection.DrsCalibration
+                key="Data"
+                outputKey="DataCalibrated"
+        />
+
+        <fact.datacorrection.RemoveSpikes
+            dataKey="DataCalibrated"
+            outputKey="DataCalibrated"
+            startCellKey="StartCellData"
+            leftBorder="${RemoveSpikes.leftBorder}"
+            spikeLimit="${RemoveSpikes.spikeLimit}"
+            topSlopeLimit="${RemoveSpikes.topSlopeLimit}"
+            maxSpikeLength="${RemoveSpikes.maxSpikeLength}"
+        />
+        <fact.datacorrection.InterpolateTimeSeries
```

Comment:
Not sure if we should interpolate the bad pixels here, or if they should rather be marked and interpolated in the analysis

Reply:
You do not need to mark anything you would just need to take the 'LONS_UnixTimeUTC' and use it to calibrate it in the 'InterpolateTimeSeries'. Actually this would make sense as we only want to calibrate with the error pixel on that day. So we could remove this here theoretically and change the analysis side of things.

Reply:
yes, I think this should be done. I think this case should be handled in https://github.com/fact-project/fact-tools/blob/2098a0fdcf4f0d3fae17e1049db1b947aead0cc9/src/main/resources/analysis/calibration.xml#L40. I think depending on the 'Creator' a different 'timeStampKey' should be used in https://github.com/fact-project/fact-tools/blob/2098a0fdcf4f0d3fae17e1049db1b947aead0cc9/src/main/resources/analysis/calibration.xml